src/Text2Image/T2IParamTypes.cs

```diff
@@ -575,8 +575,8 @@ static List<string> listVaes(Session s)
                 return s;
             }
             ));
-        BitDepth = Register<bool>(new("16Bit Depth", "Optional override for the bit depth of the image.\n If enabled, the image will be saved as a 16-bit PNG.\nIf disabled, the image will be saved as an 8-bit PNG.\n8 is the default and recommended value for most images.\n16 is only recommended for very high quality or print output.\nIf you are not using PNG format, this option is ignored.",
-            "false", IgnoreIf: "false", IsAdvanced: true, Group: GroupSwarmInternal, AlwaysRetain: true, OrderPriority: 3
+        BitDepth = Register<string>(new("Color Depth", "Specifies the color depth for PNG and TIFF formats:\n- PNG supports 8-bit and 16-bit\n- TIFF supports 8-bit, 16-bit, and 32-bit\n- Other formats ignore this setting\n\nIf 32-bit is selected for PNG, it will default to 8-bit.",
+            "8bit", GetValues: (_) => ["8bit///8-bit (256 colors)", "16bit///16-bit (65,536 colors)", "32bit///32-bit (16.7 million colors)"], IsAdvanced: true, Toggleable: true, Group: GroupSwarmInternal, OrderPriority: 3
```

Comment:
These descriptions are wrong -- "8 bit color" in this context is actually 24bit color for mathematical purposes (8 times 3 channels), there's far more than 256 colors. The description should probably read more like `8-bit per channel (24 bit total)` rather than trying to define the color count (as it's a silly high number in all cases)